### ISSUE TITLE
Add the Orderable Document List plugin to this example?

### ISSUE THREAD

**Issue by ElektrikSpark** [sanity-io/nextjs-blog-cms-sanity-v3#96]
[Orderable Document List](https://github.com/sanity-io/orderable-document-list)

Hi!

I am currently playing around with the rsc branch in the pursuit of getting Sanity Studio v3 working with the appDir for my website. I ran into some confusion around adding the Orderable Document List plugin with the current structure of adding settings.

current:
`deskTool({
      structure: settingsStructure(settingsType),
      // `defaultDocumentNode` is responsible for adding a “Preview” tab to the document pane
      defaultDocumentNode: previewDocumentNode({ apiVersion, previewSecretId }),
    }),`

Orderable Document List docs:
`structure: (S, context) => { 
                return  S.list()
                    .title('Content')
                    .items([
                        // Minimum required configuration
                        orderableDocumentListDeskItem({type: 'category', S, context}),`

I would imagine a lot of people are going to add this plugin, so having it added to this example or documented in this issue would be of great help.

Amazing job getting Sanity working with Next.js 13 and the appDir so fast. I know Sanity Product Day is coming fast so no rush on this. ❤️

**Comment by stipsan** [comment]
Great suggestion! Will definitely follow up :relaxed: And thanks so much for your feedback ✨

**Comment by stipsan** [comment]
And regarding `appDir`, have you seen #90? Might save you some time 😉 We're updating it with a workaround until Next 13's `previewData()` ships with support for static render modes in RSC.
TL;DR it currently deops to dynamic. Effectively the same thing as if you rename `getStaticProps` to `getServerProps` in Next 12. The workaround we'll be adding is using different routes for preview mode so that main routes stays static, and only preview routes are dynamic 🙇
